standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -653,6 +757,11 @@ public void abortTxns(AbortTxnsRequest rqst) throws NoSuchTxnException, MetaExce
               (txnids.size() - numAborted) +
               " transactions have been aborted or committed, or the transaction ids are invalid.");
         }
+
+        for (Long txnId : txnids) {
```

Comment:
This method can lead to partially aborted txns too based on above check but we don't have a mechanism to see which one failed. So, if we log event, then it may lead to failure in target as well for repl load.
Currently, we don't have a fix for this. Create a separate JIRA mentioning that "ABORT TRANSACTIONS" command doesn't fail if abort of all txns in the input list was not successful. Also cc to Eugene.